scene/gui/tree.cpp

```diff
@@ -3513,7 +3525,17 @@ void Tree::_update_all() {
 }
 
 Size2 Tree::get_minimum_size() const {
-	return Size2(1, 1);
+	if (h_scroll_enabled && v_scroll_enabled) {
+		return Size2();
```

Comment:
What happened to `Size2(1, 1)`? Does this change any behaviour elsewhere unexpectedly? Or is this desired?

Reply:
This is desired, it does not make any sense to keep a minimum size of (1, 1)

Reply:
I understand, I am just worried about divisions by zero elsewhere in the code that has relied on this previous minimum size behaviour or some such.

Reply:
That's unlikely, a lot of other Control nodes return Size2() when they do not have a minimum size. Which is the case here.